index.html

```diff
@@ -28,19 +28,192 @@ <h1 class="introduction">I'm Emmanuel, glad to see you</h1>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+
+    <div class="my-works">
+        <figure>
+            <img src="images/multi-post-stories-2-project.jpg" class="works-image" alt="Tonic App" width="100%"> 
+        </figure>
+        <div class="works-description">
+            <h3 class="my-works-project-title">Multi-Post
+                Stories</h3>
+            <ul class="works-headers">
+                <li class="title-header">CANOPY</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">Back End Dev</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">2015</li>
+            </ul>
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+    <!--My projects end//-->
+
+    <!-- About Myself Start-->
+    <section class="headline">
+        <h1 class="introduction">About Myself</h1>
 
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look 
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t 
+            hesitate to contact me.</p>    
+        <h4 class="connect">LETS CONNECT</h4>
         <ul class="social">
             <li><a href="#"><img src="images/twit.png" alt="Twitter"></a></li>
             <li><a href="#"><img src="images/Vector.png" alt="Linkedin"><img src="images/in.png" alt=""></a></li>
             <li><a href="#"><img src="images/m.png" alt="Mi"></a></li>
             <li><a href="#"><img src="images/cat.png" alt="Guthub"></a></li>
             <li><a href="#"><img src="images/peace.png" alt="Alison"></a></li>
         </ul>
+        <section class="project-more" style="margin-left: 14px;">
+            <button class="works-see-more"><span class="more-text">Get my resume</span></button>
```

Comment:
Kindly modify the margins and padding of the `get resume` button to match your chosen design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192076614-adaaa333-2cca-4dff-8927-5e740dd7c84f.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/192076620-e4f6c91f-708b-44fc-871f-d70d02b7e645.png)